microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -39,13 +39,18 @@ See [Payment Request API](https://developer.mozilla.org/docs/Web/API/Payment_Req
 ## Checking whether the Digital Goods API is available
 
 To detect whether you've correctly enabled the API on your website, check for the `getDigitalGoodsService` method in the window object:
```

Comment:
this is phrased as if the dev needs to take action to enable the API, but we're not telling them what action that is.
is there an origin trial?
is there a flag to enable somewhere?
does this come with a pre-release version of Edge?

@runyuany

Reply:
Yes, there is a running origin trial for this feature now, but we are planning to graduate the feature from OT soon. Do you think we should list the OT setup guide here for now?
No, the flag will be enabled by default with the feature shipping.
Yes, this will come with pre-release of Edge. I'll provide the exact version number here when we got one.